docs/tutorials/custom-commit-notifications.md

```diff
@@ -0,0 +1,124 @@
+Things happen all the time in your development workflow. Commits are pushed,
+builds are started, they fail or succeed. Team members, or even users, open and
+comment issues everyday.
+
+All that flow of information can become overwhelming. With Atomist, you can
+attach behavior to these events and automate all sorts of near real-time
+responses.
+
+These tutorials show you how to rely on Atomist to automate those scenarios.
+
+!!! tip "All you need is Rug"
+    These tutorials aim at showing you the code that you could write to
+    automate event reactions. They assume you have a [Rug project][ugpj]. You
+    can easily add [Rug event handler][rugev] to that project via the public 
+    [AddTypeScriptEventHandler Rug editor][rugeditor] provided by Atomist.
+
+[rugev]: /user-guide/rug/handlers.md
+[ugpj]: /user-guide/rug/projects.md
+[rugeditor]: https://github.com/atomist/rug-editors#addtypescripteventhandler
+
+## Getting Notified from a User Commit
+
+The first events you will ask Atomist to react to are commits pushed by a user.
+To achieve this, you add a [Rug event handler][rugev] to your
+[Rug project][ugpj] that, once you have [published][ugpub] it to your team, will
+be evaluated and executed by Atomist on such events.
+
+[ugpub]: /user-guide/rug/lifecycle.md#publishing
+
+Below is such an event handler:
+
+```typescript linenums="1"
+{!../../.atomist/handlers/event/NewCommitPushedToGeneral.ts!}
+```
+
+An event handler tells Atomist what it is interested in via its
+[path expression][ugpxe] on line 10. Notice in particular how the handler
+filters only commit made by a given author on line 11 (here, Alice's commits). 
+Finally, line 12 asks Atomist to also feed the handler with the commits
+repository details.
+
+[ugpxe]: /user-guide/rug/path-expressions.md
+
+Once it gets called, the event handler builds a message to send to the 
+`#general` channel of your team. 
+
+## Message me on Commits
+
+In the previous section, you wrote an event handler that sends a message to
+the general channel of your team when Alice pushes her commits.
+
+Let's see now you could ask Atomist to send a direct message instead.
+
+```typescript linenums="1" hl_lines="22"
+{!../../.atomist/handlers/event/NewCommitPushedToDM.ts!}
+```
+
+As you can see, the only change is on line 22 where you provide the name of 
+the user to send the message to, here `@Bob`.
+
+But wait, you can have it both ways and send a direct message and a 
+channel message:
+
+```typescript linenums="1" hl_lines="21 26 31 33"
+{!../../.atomist/handlers/event/NewCommitPushedToDMAndGeneral.ts!}
+```
+
+Event handlers can dispatch multiple messages from a single event as shown 
+on the highlithed lines in this example.
+
+## Fetch and Display the Modified Files 
+
+Sending messages on events is critical for information flow but what if you 
+could perform actions on those events! On a commit, it should be handy to see 
```

Comment:
You are still just sending a message with this handler, albeit a message with more information, so the first sentence here is a bit misleading. The first sentence could be "Sending simple commit notifications is useful, but can we provide more context so people can get all the information they need without leaving Slack and go to GitHub.com?"